src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py

```diff
@@ -141,6 +141,17 @@ def queueNewRequests(self, queue):
             self.logger.info("%s element(s) obtained from RequestManager" % work)
         return work
 
+    def _handleAbortedAndForceCompleted(self, queue, ele, state):
+        if state == 'aborted':
+            nextState = 'aborted-completed'
+        elif state == "force-complete":
+            nextState = 'completed' 
+        
+        if ele['Status'] in ['Canceled', 'Done']:
+            self.reportRequestStatus(ele['RequestName'], nextState)
```

Comment:
This element is GQ workqueue inbox element so there is only one element per request.